client/src/main/java/org/apache/rocketmq/client/consumer/DefaultMQPushConsumer.java

```diff
@@ -401,6 +408,14 @@ public void setPullThresholdForQueue(int pullThresholdForQueue) {
         this.pullThresholdForQueue = pullThresholdForQueue;
     }
 
+    public int getPullThresholdForTopic() {
+        return pullThresholdForTopic;
+    }
+
+    public void setPullThresholdForTopic(int pullThresholdForTopic) {
+        this.pullThresholdForTopic = pullThresholdForTopic;
```

Comment:
I guess setting `pullThresholdForTopic` to 0 or negative wouldn't make sense here. How about warning a user about it?